tests/unit/setup-context-test.js

```diff
@@ -79,6 +79,21 @@ module('setupContext', function(hooks) {
     test('it calls setContext with the provided context', function(assert) {
       assert.equal(getContext(), context);
     });
+
+    test('can be used for unit style testing', function(assert) {
+      context.owner.register(
+        'service:foo',
+        Service.extend({
+          someMethod() {
+            return 'hello thar!';
+          },
+        })
+      );
+
+      let subject = context.owner.lookup('service:Foo');
+
+      assert.equal(subject.someMethod(), 'hello thar!');
+    });
```

Comment:
should we test `this.inject('foo')` here too?

Reply:
Well, that API is no longer present (see examples and reasoning in the RFC), but I will add an example to both these tests and the rendering tests that will be a better demo of the go forward way to do it...